### PR TITLE
bug fix for Issue #312 windows certs

### PR DESCRIPTION
**Pull request by dwk2**
- This bug fix creates a Windows alternative for running the `/scripts/proxy/ssl-certs/Local.sh` and `/scripts/proxy/ssl-certs/Demo.sh` files as these fail because Windows chokes on un-escaped characters.
- Created two new files named `local-windows-only.sh` and `demo-windows-only.sh`
- The new files correct the un-escaped characters as per this thread:
[https://stackoverflow.com/questions/31506158/running-openssl-from-a-bash-script-on-windows-subject-does-not-start-with](https://stackoverflow.com/questions/31506158/running-openssl-from-a-bash-script-on-windows-subject-does-not-start-with)
- Updated the documentation for both `install-local-new` and `install-local-migrate` 
- Also added a requirement to `install-local-new` and `install-local-migrate` that Windows users must use Git for Windows for two reasons:
    - Git Bash command line works where Powershell fails
    - Git for Windows installs openssl.exe, which the Local.sh will utilize to generate the openSSL certs.

### CODE REVIEW

**Comment by marksandford** [comment]
There was a very small conflict preventing merge, which I resolved (since it was my fault these PRs sat for days!)

File: docs/install/install-local-migrate.md [hunk 1]
```diff
@@ -298,22 +300,27 @@ You can reuse some of the older Production settings in the `local.env` if you li
 
 ---
 
-## Step 6: Create New Self-Signed Certs for Your Project
-**This step is optional, and does not currently work on Windows.**
+## Step 6: Create new self-signed certs for your project
```
Comment (bseeger) [inline docs/install/install-local-migrate.md]: Can you bring the caps back, or change all of them? :)  Thanks!

**Comment by bseeger** [comment]
I wonder if there is someone else in the community who can try this flow out on a windows machine...?

File: docs/install/install-local-migrate.md (hunk 1 above)
Comment (dwk2) [reply docs/install/install-local-migrate.md]: @bseeger - I'll add the initial-caps consistency back in my next docs PR. - dkc

**Comment by dwk2** [comment]
Thank you @marksandford !
@bseeger - Yes, RPI is ready to try use a Windows machine as they build their Local environment.

**Pull request merged**